src/node.cc

```diff
@@ -1108,33 +1135,43 @@ InitializationResult InitializeOncePerProcess(
     OPENSSL_INIT_free(settings);
 
     if (ERR_peek_error() != 0) {
-      result.exit_code = ERR_GET_REASON(ERR_peek_error());
-      result.early_return = true;
-      fprintf(stderr, "OpenSSL configuration error:\n");
-      ERR_print_errors_fp(stderr);
+      // XXX: ERR_GET_REASON does not return something that is
```

Comment:
I grepped under doc/contributing for XXX and TODO to see if we've documented when to use one versus the other but could not find any guidance. Is it documented somewhere?

Reply:
To me, `XXX` is “this is odd (and there might be a better alternative)” and `TODO` is “there is definitely a better alternative”. Ultimately, not a big difference.

Reply:
@addaleax thanks, if its not documented anywhere I might PR that info our contributing docs somewhere. Does that make sense to you?

Reply:
@mhdawson Sure, go for it! I think it's also fine to adjust the definitions if other collaborators feel that these (should) have different meanings.